src/js/components/DataClearFilters/__tests__/DataClearFilters-test.tsx

```diff
@@ -73,6 +73,15 @@ describe('DataClearFilters', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  test('renders outside grommet wrapper', () => {
+    const { asFragment } = render(
+      <Data data={data}>
+        <DataClearFilters />
+      </Data>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Should we remove this test since DataClearFilters is wrapped by Data?

Reply:
yes must of missed doing now!